qiime2/sdk/parallel_config.py

```diff
@@ -13,14 +13,35 @@
 import tomlkit
 import threading
 import importlib
+import pkg_resources
 
 from parsl.config import Config
 
-
 PARALLEL_CONFIG = threading.local()
 PARALLEL_CONFIG.parallel_config = None
 PARALLEL_CONFIG.action_executor_mapping = {}
 
+# We write a default config to a location in the conda env if there is an
```

Comment:
Looks like these are the same changes as #696 - should this be separated?

Reply:
Good catch. I'll remove the comment and let it come from the other PR because that functionality isn't addressed here. The way the module scope variables are created here is fine regardless though